rental.rb

```diff
@@ -1,11 +1,12 @@
-class Rentals
-  attr_accessor :date, :person, :book
+class Rental
+  attr_accessor :date, :book, :person
 
-  def initialize(date)
+  def initialize(date, book, person)
     @date = date
-    @person = person
-    person.rental_data << self
     @book = book
-    book.rental_data << self
+    @person = person
+
+    @person.add_rental(self)
+    @bookk.add_rental(self)
```

Comment:
- [ ] An error occurs when trying to create a add a new rental as shown below 👇 Please try to fix it
![image](https://user-images.githubusercontent.com/46839548/213865186-247d95d3-05a9-4971-aadd-5b21291827e5.png)